README.md

```
@@ -12,13 +12,13 @@ To run the docs site locally, follow these steps:
   * Install Node
   * Install Yarn
 
-2. Clone this repository and make it your working directory.
+1. Clone this repository and make it your working directory.
 
-3. Run `yarn install` to install all dependencies.
+2. Run `yarn install` to install all dependencies.
 
-4. If this is the first time running the site, run `yarn run develop:env-setup` to setup your [environment variables](#environment-variables). Visit `localhost:8000` to view the site.
+3. If this is the first time running the site, run `yarn run develop:env-setup` to setup your [environment variables](#environment-variables). Visit `localhost:8000` to view the site.
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)